privacy_evaluator/attacks/membership_inference/membership_inference_analysis.py

```diff
@@ -16,15 +16,20 @@ class MembershipInferenceAttackAnalysis:
     """Represents the membership inference attack analysis class."""
 
     def __init__(
-        self, attack_type: Type[MembershipInferenceAttack], input_data: AttackInputData
+        self,
+        attack_type: Type[MembershipInferenceAttack],
+        input_data: AttackInputData,
+        **kwargs,
```

Comment:
Are these `kwarg` needed here? In line 65 it is alredy hardcoded that `probabilities=True` thus these `kwargs` won't  change anything with respect to the returned values of the `infere` method. Or are these `kwargs` needed for something else?

Reply:
They're actually not because of the probabilities here. In the original implementation I forgot that the attacks also might get arguments, e.g. black-box you can choose between "nn", "rf" and "gb" and this is the way to pass them when instantiating the attack object in the `analysis` method. We could also change the API here as per your suggestions in the original PR.

Reply:
Ah ok, that's totally fine.